### PR TITLE
8273280: Update README.md and CONTRIBUTING.md for jfx17u

### PR DESCRIPTION
**Pull request by kevinrushforth**
This PR updates [`README.md`](https://github.com/openjdk/jfx17u/blob/3082a845253dd00d46d36500abc42464459e59ed/README.md) and [`CONTRIBUTING.md`](https://github.com/openjdk/jfx17u/blob/3082a845253dd00d46d36500abc42464459e59ed/CONTRIBUTING.md) to be suitable for update releases. The two files were copied from jfx11u, with the only changes being to replace "11" with "17" in the two files.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8273280](https://bugs.openjdk.java.net/browse/JDK-8273280): Update README.md and CONTRIBUTING.md for jfx17u

### Reviewers
 * [Johan Vos](https://openjdk.java.net/census#jvos) (@johanvos - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jfx17u pull/1/head:pull/1` \
`$ git checkout pull/1`

Update a local copy of the PR: \
`$ git checkout pull/1` \
`$ git pull https://git.openjdk.java.net/jfx17u pull/1/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1`

View PR using the GUI difftool: \
`$ git pr show -t 1`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jfx17u/pull/1.diff">https://git.openjdk.java.net/jfx17u/pull/1.diff</a>

</details>

### CODE REVIEW

**Comment by kevinrushforth** [comment]
@johanvos can you review?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,227 +1,14 @@
...
-If you are changing anything that might possibly affect rendering, you should run a full test with robot enabled:
-
-```sh
-bash ./gradlew -PFULL_TEST=true -PUSE_ROBOT=true all test
+Backport LONG-COMMIT-HASH
 ```
 
-Even more documentation on OpenJFX projects and its build system can be found on the
-[OpenJFX Wiki](https://wiki.openjdk.java.net/display/OpenJFX/).
+where `LONG-COMMIT-HASH` is the long (40 char) commit hash of the fix as found in the main jfx repo. The Skara tooling will then note that it is a backport, and replace the title with the correct issue title.
```
Comment (johanvos) [inline CONTRIBUTING.md:14]: Should the next parts of the process be explained as well (e.g. optional review if no clean patch, merging?)

Comment (kevinrushforth) [reply CONTRIBUTING.md:14]: Good idea, but I'd prefer to do that as a follow-on fix; that change could then be backported to jfx11u (as a clean backport).

Comment (johanvos) [reply CONTRIBUTING.md:14]: makes sense

**Review by johanvos — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (kevinrushforth) [reply CONTRIBUTING.md:14]: I filed https://bugs.openjdk.java.net/browse/JDK-8273732 as a follow-up.

**Pull request closed without merging**